web/src/components/Button.tsx

```diff
@@ -20,6 +20,7 @@ export const buttonInnerRing = cva(
 					"group-active:border-cyan-600",
 				],
 				secondary: ["border-white/25"],
+				fixed: ["border-white/25"],
```

Comment:
why do we need this?
is "fixed" a variant on buttons?
If you want to extend a styling behavior of a cva component, use
for example:
`button({intent: "primary", className: "your-extra-class"})`
we do not need to have another variant here